src/modules/hip/kernel/warp_perspective.hpp

```diff
@@ -0,0 +1,461 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+// -------------------- Set 0 - warp_perspective device helpers --------------------
+
+__device__ void warp_perspective_srclocs_hip_compute(float perspectiveMatrixElement, float4 locHomComponent_f4, float4 roiComponent_f4, d_float8 locHomW_f8, d_float8 *locSrcPtr_f8)
```

Comment:
we can pass this parameters locHomComponent_f4, roiComponent_f4, locHomW_f8 as pointers?
```suggestion
__device__ void warp_perspective_srclocs_hip_compute(float perspectiveMatrixElement, float4 *locHomComponent_f4, float4 *roiComponent_f4, d_float8 *locHomW_f8, d_float8 *locSrcPtr_f8)
```

Reply:
Updated, locHomW_f8 alone made pointer

Reply:
Others avoided due to casting